{"name": "Ошибка при получении заказов неавторизованным пользователем", "status": "passed", "start": 1733686230913, "stop": 1733686231335, "uuid": "8e8160e2-d335-49da-a0f7-aea1b7942cf9", "historyId": "6261750354a184854c9dd27d87fd5a69", "testCaseId": "6261750354a184854c9dd27d87fd5a69", "fullName": "tests.test_get_user_order.TestGetOrders#test_get_orders_without_authorization", "labels": [{"name": "feature", "value": "Orders"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_user_order"}, {"name": "subSuite", "value": "TestGetOrders"}, {"name": "host", "value": "Karinas-MacBook-Pro.local"}, {"name": "thread", "value": "18924-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_user_order"}]}